public/javascripts/Progress/src/Progress.js

```diff
@@ -51,22 +51,26 @@ function Progress (_, $, userRole) {
         });
     }
 
-    function putUserOrg(e) {
+    function putUserTeam(e, newTeam) {
         var parsedId = $(this).attr('id').split("-"); // the id comes in the form of "from-startOrg-to-endOrg"
-        var startOrg = parsedId[1];
-        var endOrg = parsedId[3];
+        var startTeam = parsedId[1];
+        if(newTeam != null){
+            var endTeam = newTeam;
+        } else {
+            var endTeam = parsedId[3];
+        }
         $.ajax({
             async: true,
-            url: '/userapi/setUserOrg/' + endOrg,
+            url: '/userapi/setUserOrg/' + endTeam,
             type: 'put',
             success: function (result) {
                 window.location.reload();
-                if (endOrg != startOrg) {
```

Comment:
Something isn't working quite right here.
* If I have no team and create a new one, it logs `Click_module=create_team=`. Missing the team id! Plus I think that it should  probably log both that you created it AND that you joined it!
* Joining a team doesn't log anything
* Switching team logs the leaving of a team correctly, but not the joining of the new team

But leaving a team is logged correctly!

Not sure how much of this was broken before you got here, but we've gotta fix it as we see it rather than adding more broken pieces on top!